src/convertToObject.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes string with styles
- * and returns object where CSS properties are keys
- * and values are the values of related CSS properties
-
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrStyles = sourceString.split(';').map(style => {
+    return style.trim();
+  }).filter(style => style.length > 0);
+
+  const result = arrStyles.reduce((prev, current) => {
```

Comment:
You can also call this `reduce` in cahin with previous methods.
And, you create result variable and return it right away.